### PR TITLE
Added float types and float slices

### PR DESCRIPTION
**Pull request by caledhwa**
Hello!

We found that in our project we really wanted to represent floats in our env config. Therefore, here's a pull request to do just that. I implemented both float32 and float64 in addition to slice types. Unit tests and readme updated. Thanks!

### CODE REVIEW

File: env_test.go [hunk 1]
```diff
@@ -21,6 +21,10 @@ type Config struct {
 	Numbers     []int         `env:"NUMBERS"`
 	Bools       []bool        `env:"BOOLS"`
 	Duration    time.Duration `env:"DURATION"`
+	Float32			float32 			`env:"FLOAT32"`
+	Float64			float64 			`env:"FLOAT64"`
+	Float32s  	[]float32			`env:"FLOAT32S"`
+	Float64s	  []float64			`env:"FLOAT64S"`
```
Comment (caarlos0) [inline env_test.go]: gofmt seems to be missing here

Comment (caledhwa) [inline env_test.go]: DOH! Of course! Let me get that. Thanks for catching!

**Comment by caarlos0** [comment]
Thanks!

**Comment by dselans** [comment]
👍

**Comment by caledhwa** [comment]
No problem. Glad to help!

**Pull request merged**